### ISSUE TITLE
Divide and conquer synthesis

### ISSUE THREAD

**Issue by kelenelee** [hackforla/design-systems#431]
# BACKGROUND

The **[previous research team members completed affinity diagrams](https://github.com/hackforla/design-systems/issues/279)** for the **[interview study](https://github.com/hackforla/design-systems/milestone/7)**, but not all the **[stickies](https://www.figma.com/file/rbW89yoCha3ojI7kxaqvil/Interviews---Affinity-Diagram?node-id=0%3A1)** are up to affinity diagramming standards.

I have directed the current research team to **redo analysis**, so that synthesis is more successful, so that we can extract meaningful, helpful, and actionable recommendations for HfLA’s practices as a whole.

In order to create a new, better analysis strategy, I had to rewrite the **[original research plan](https://docs.google.com/document/d/1peRWmKWLpYEjqtlszfqmSSpsPm95qagnsFCJdiyereo)** into a **[second research plan](https://docs.google.com/document/d/1q9BdkerP6zwT0_TeVWqUwO6KI7YblVeU0OTORIJFxZ0)**. The original plan's objectives and directives were too unclear that the other researchers could not effectively read and absorb its importance. **It's important that the researchers read the new plan so that the study background is clear to everyone.**

### change log

- **5/24** @hdchangie @LeslieCi @allthatyazz @kelenelee discussed this task during the research team meeting
- **5/26** @hannahcarlan and @kelenelee had a Slack chat to clarify the action items and research goals. The research objectives were not written clearly in the plan. The **[plan has been updated](https://docs.google.com/document/d/1q9BdkerP6zwT0_TeVWqUwO6KI7YblVeU0OTORIJFxZ0/edit#)** with clearer research objectives that can translate into themes for analysis. In order to preserve the old plan, I decided to make a new doc instead of editing the original one. 
  - As for analysis methodology - I was worried about asking the researchers to do the affinity diagramming technique. We are not trained in it by an expert and poor QC is why we have to redo it. But, because synthesis down the line will be difficult if everyone's analysis methods are NOT the same, I am asking that everyone watch a video about affinity diagram best practices from a Delta CX tutorial.
- **5/31** @kelenelee updated the GitHub issue to clarify new updates from 5/26. Based on the current state of progress, I don't think it's wise to pivot back to the Figma Kit by 6/6. The researchers have put in a lot of time to rewatching the interview videos, and if we take a break now before we complete synthesis, then we may have to do repeat work whenever we come back to this study. Furthermore, if there are findings that are relevant to the Kit, then we should find those first before continuing with the Kit study.
- **6/2** @kelenelee removed P1 from this iteration of synthesis. Because there's only one dev in this study amongst the multiple designers and product managers, it's not clear what value synthesis of P1 will bring.
- **6/13** @kelenelee created a new issue specifically for interview transcription: #434 
- **6/14** P5 and P8 are de-prioritized from analysis because they were not officially their team's PM
- **7/18** @hdchangie presented a slide deck on designer participants
- **7/20** Transcripts were moved to [google docs](https://drive.google.com/drive/u/1/folders/1G5uOImuMjhaWmc60p-wZ9c2PPLLbUXqV) instead of [spreadsheets](https://docs.google.com/spreadsheets/d/1Ma_nPqdCMM7Weto_Zgrp7rlbJaT5ainPCZuIM2UcejI) to allow for coding via comments and highlights. P5 and P8 were coded
- **8/1** @kelenelee presented an initial slide deck summarizing just PM participants

# ACTION ITEMS

### Designers

- [x] P10 @hannahcarlan 
- [x] P11 @LeslieCi 
- [x] P4 @hannahcarlan or @LeslieCi 
- [x] Meet to collaborate on synthesis of designer data
- [x] Put together some kind of written summary

### Product Managers
To preserve the old Step 2, re-analysis should be done in **Step 4**.

- [x] P3 @kelenelee 
- [x] P6 @kelenelee 
- [x] P9 @kelenelee 
- [x] P2 @hdchangie 
- [x] P7 @hdchangie 
- [x] Meet to collaborate on synthesis of PM data
- [x] Put together some kind of written summary

~P8~ @cunninghamej 
~P5~

# SUBTASKS 
ALL ASSIGNEES PLEASE MAKE A COMMENT WITH THIS TEMPLATE

To encourage our team members in the habit of documenting their thought process, the template includes questions that you need to answer by editing the comment. I know that editing in Markdown can be a pain (sorry!) but I hope that the spaces will help the typing experience be less overwhelming. Let me know how my experiment goes for you.

```
- [ ] Read the **[updated plan](https://docs.google.com/document/d/1q9BdkerP6zwT0_TeVWqUwO6KI7YblVeU0OTORIJFxZ0)** 
- [ ] For learning about the **affinity diagramming technique**, you should watch **[this series of videos](https://www.youtube.com/playlist?list=PLPzC8RwtTOZkt7lukZU0pTZSw6D2Uok2H)** on a UXR study of home cooks. It is one of the only series where a senior UX person actually demonstrates UXR techniques. You can skip to the affinity diagramming video, but the other videos in the playlist can give you more context for the study. (The mini-series is part of the greater **[Micro-Lessons series](https://www.youtube.com/playlist?list=PLSaGB80mfUaSdlERUdhhQSbTBw214xWjS)** which includes resources on personas, surveys, coding, and more.)
- [ ] For any **[interviews recordings](https://drive.google.com/drive/folders/1G5uOImuMjhaWmc60p-wZ9c2PPLLbUXqV)** you rewatch, **clean up the transcript**. Add timestamps and remove typos and "um"s. If you have time and you're already there watching the video, this would help us quite a bit.
- [ ] For your assigned participants, **redo the affinity diagram** and use the **[updated plan's list of themes](https://docs.google.com/document/d/1q9BdkerP6zwT0_TeVWqUwO6KI7YblVeU0OTORIJFxZ0/edit#bookmark=id.ikcsa4d2na0o)** to guide you.
- [ ] Come up with **meaningful and helpful insights** that you can present to the rest of the research team
- [ ] Come back to your comment and **answer the questions** below
- [ ] Scroll back up to the main part of the issue and **check off your assigned tasks**.
___
### Any adjustments to the tasks you were originally assigned, or problems you ran into?



### Any significant discoveries, successes? New ideas for future research? 



### Any suggestions for affinity diagramming resources from experienced UX researchers? 



### Any other suggestions for improving your GitHub experience for this issue, or for GitHub overall?


```

**Comment by kelenelee** [comment]
### Some notes: 
Yep, the GitHub template is looking a little different than usual. To get our team members in the habit of talking more about their thought process, I'm including questions with spaces in the template.

The affinity diagramming video is an hour-long video by my mentor and 20-year experienced UX researcher and designer, Debbie Levitt.

**Comment by kelenelee** [comment]
- [ ] Read affinity diagramming tips 
- [ ] Watch **[this video on affinity diagramming](https://youtu.be/1Ygck9csiYQ)**
- [ ] For any interviews recordings you rewatch, clean up the transcript. Add timestamps and remove typos and "um"s. This is optional but IF you have time and you're already there watching the video, this would help us quite a bit
- [ ] When you complete the analysis for the participants you're assigned to, scroll back up to the main part of the issue and check off your name.
- [ ] Come back to your comment and answer the questions below

## Any adjustments to the tasks you were originally assigned?



## Any significant discoveries, successes? New ideas for future research? 



## Any problems you ran into?



## Any suggestions for affinity diagramming resources from experienced UX researchers? 



## Any ideas for how to make the directions in this GitHub issue clearer for you?



## Any other suggestions for improving your GitHub experience?

**Comment by hdchangie** [comment]
- [ ] Read affinity diagramming tips 
- [ ] Watch **[this video on affinity diagramming](https://youtu.be/yUqpS1C1bG4)**
- [ ] For any interviews recordings you rewatch, clean up the transcript. Add timestamps and remove typos and "um"s. This is optional but IF you have time and you're already there watching the video, this would help us quite a bit
- [ ] When you complete the analysis for the participants you're assigned to, scroll back up to the main part of the issue and check off your name.
- [ ] Come back to your comment and answer the questions below

### Any adjustments to the tasks you were originally assigned, or problems you ran into?



### Any significant discoveries, successes? New ideas for future research? 



### Any suggestions for affinity diagramming resources from experienced UX researchers? 



### Any other suggestions for improving your GitHub experience for this issue, or for GitHub overall?

**Comment by hannahcarlan** [comment]
- [ ] Read affinity diagramming tips 
- [ ] Watch **[this video on affinity diagramming](https://youtu.be/yUqpS1C1bG4)**
- [ ] For any interviews recordings you rewatch, clean up the transcript. Add timestamps and remove typos and "um"s. This is optional but IF you have time and you're already there watching the video, this would help us quite a bit
- [ ] When you complete the analysis for the participants you're assigned to, scroll back up to the main part of the issue and check off your name.
- [ ] Come back to your comment and answer the questions below

### Any adjustments to the tasks you were originally assigned, or problems you ran into?



### Any significant discoveries, successes? New ideas for future research? 



### Any suggestions for affinity diagramming resources from experienced UX researchers? 



### Any other suggestions for improving your GitHub experience for this issue, or for GitHub overall?

**Comment by cunninghamej** [comment]
- [ ] Read the **[updated plan](https://docs.google.com/document/d/1q9BdkerP6zwT0_TeVWqUwO6KI7YblVeU0OTORIJFxZ0)** 
- [ ] For learning about the **affinity diagramming technique**, you should watch **[this series of videos](https://www.youtube.com/playlist?list=PLPzC8RwtTOZkt7lukZU0pTZSw6D2Uok2H)** on a UXR study of home cooks. It is one of the only series where a senior UX person actually demonstrates UXR techniques. You can skip to the affinity diagramming video, but the other videos in the playlist can give you more context for the study. (The mini-series is part of the greater **[Micro-Lessons series](https://www.youtube.com/playlist?list=PLSaGB80mfUaSdlERUdhhQSbTBw214xWjS)** which includes resources on personas, surveys, coding, and more.)
- [ ] For any **[interviews recordings](https://drive.google.com/drive/folders/1G5uOImuMjhaWmc60p-wZ9c2PPLLbUXqV)** you rewatch, **clean up the transcript**. Add timestamps and remove typos and "um"s. If you have time and you're already there watching the video, this would help us quite a bit.
- [ ] For your assigned participants, **redo the affinity diagram** and use the **[updated plan's list of themes](https://docs.google.com/document/d/1q9BdkerP6zwT0_TeVWqUwO6KI7YblVeU0OTORIJFxZ0/edit#bookmark=id.ikcsa4d2na0o)** to guide you.
- [ ] Come up with **meaningful and helpful insights** that you can present to the rest of the research team
- [ ] Come back to your comment and **answer the questions** below
- [ ] Scroll back up to the main part of the issue and **check off your assigned tasks**.
___
### Any adjustments to the tasks you were originally assigned, or problems you ran into?



### Any significant discoveries, successes? New ideas for future research? 



### Any suggestions for affinity diagramming resources from experienced UX researchers? 



### Any other suggestions for improving your GitHub experience for this issue, or for GitHub overall?

**Comment by kelenelee** [comment]
At the 6/6 all-hands meeting I presented what we've found so far with the PM analysis.

- So far no personas yet - Still working on P3 and P8 - consistent characteristics across participants so far
- Participants are junior-level career transitioners coming to HfLA to learn new skills 
- Low product management skill level, even in participants with prior experience in the  workplace and in product development. This leads to suboptimal product development workflows. 
  - Documentation and action are competing for volunteer’s time
  - Teams have no UXR or market research to determine product-market fit 
- Design and dev teams don’t collaborate because the team is not at a building stage yet

As for this study's methodology,

- We don’t have the expertise to evaluate PM skill set
- Contextual inquiry would have been a more effective method than self-reported interview data

Notes on what has been found about the designers so far

- No emphasis on documentation - move fast and get the work done
- Mismatch between stakeholder wants and what people want
- Lack of mentorship from seniors - more peer “mentoring”
- People’s goals for joining HfLA - how to connect that to documentation? How to INCENTIVIZE newbies?

Feedback from Yas

- Specify “lack of documentation.” Improve the documentation - how ??? it’s not a linear process at all - things need to be done and redone - WHAT should be documented? How much? Documenting everything would be difficult / impossible
- What is documentation vs proper workflow?
- Can interviews from one group solve something from the other group?

**Comment by kelenelee** [comment]
6/7 research meeting was cancelled. 
6/16 research meeting, most of it was a group session of PM synthesis with Eric, Dongchen and me in the **[affinity diagram](https://www.figma.com/file/rbW89yoCha3ojI7kxaqvil/Interviews---Affinity-Diagram?node-id=0%3A1)**. Leslie and Hannah have already finished their affinity diagrams for the designer participants and are now working on a **[list of insights in this doc](https://docs.google.com/document/d/1i1M7XVclYGXn1-90LZCyCfJaUEA0htfDx6WlBqF5wC0/edit#)**.

**Comment by kelenelee** [comment]
6/16 @hdchangie and I met up to for another working session. We decided that P8 should be left out of the synthesis for now, along with P5, because those participants had PM responsibilities but were not PMs in title.

After wrapping up the stickies transfer that we didn't finish on 6/14, we started looking at individual sections of stickies to look for patterns, and **[transferring our insights into a doc](https://docs.google.com/document/d/1zEK7z1vV3NT0FKMA7zsvGpIHfJ75XNElI0kcRs237AA/edit)**.

We will continue this process for each section of stickies.
